homeassistant/components/device_tracker/fritz.py

```diff
@@ -0,0 +1,94 @@
+""" Supports scanning a FritzBox router"""
+import logging
+from datetime import timedelta
+from homeassistant.const import CONF_HOST, CONF_USERNAME, CONF_PASSWORD
+from homeassistant.helpers import validate_config
+from homeassistant.util import Throttle
+from homeassistant.components.device_tracker import DOMAIN
+
+
+# Return cached results if last scan was less then this time ago
+MIN_TIME_BETWEEN_SCANS = timedelta(seconds=5)
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['fritzconnection==0.4.6']
+
+def get_scanner(hass, config):
+    """ Validates config and returns FritzBoxScanner obj"""
+    if not validate_config(config,
+                           {DOMAIN: [CONF_HOST, CONF_USERNAME, CONF_PASSWORD]},
```

Comment:
It seems that the FritzBoxScanner requires no configuration - this can be removed.